pkg/querier/stats/stats.proto

```diff
@@ -39,4 +39,10 @@ message Stats {
   // The total size of postings touched in store gateway for a specific query, in bytes.
   // Only successful requests from querier to store gateway are included.
   uint64 store_gateway_touched_posting_bytes = 12;
+  // The total number of samples scanned while evaluating a query.
+  // Equal to TotalSamples in https://github.com/prometheus/prometheus/blob/main/util/stats/query_stats.go
+  uint64 processed_samples = 13;
```

Comment:
I am not sure I understand where the word "processed" comes from. Can you explain?

Maybe scanned_samples sounds better?

Reply:
`Processed` means the total number of considered samples when the query is executed.
But now, I feel like `scanned` is more intuitive to understand. Thanks.